community/io/src/main/java/org/neo4j/io/pagecache/impl/SingleFilePageSwapperFactory.java

```diff
@@ -100,4 +137,54 @@ public long getRequiredBufferAlignment()
     {
         return 1;
     }
+
+    private class WrappingFileHandle implements FileHandle
```

Comment:
It can be, but it's an implementation detail of the `PageSwapperFactory`. If something needs more unit testing it would be the `PSF`.